modules/system/console/OctoberUtil.php

```diff
@@ -294,8 +295,46 @@ protected function utilPurgeUploads()
         if (!$this->confirmToProceed('This will PERMANENTLY DELETE files in the uploads directory that do not exist in the "system_files" table.')) {
             return;
         }
-
-        // @todo
+        
+        $totalCount = 0;
+        $uploadsPath = Config::get('filesystems.disks.local.root', storage_path('app'));
+        $uploadsPath .= '/uploads';
+        
+        /*
+         * Recursive function to scan the directory for files and ensure they exist in system_files.
+         */
+        $purgeFunc = function ($targetDir) use (&$purgeFunc, &$totalCount) {
+            if ($files = File::glob($targetDir.'/*')) {
+                foreach ($files as $file) {
+                    if (!is_file($file)) {
+                        continue;
+                    }
+                    
+                    if (FileModel::where('disk_name', basename($file))->count()) {
+                        continue;
+                    }
+                    
+                    $this->info('Purged: '. basename($file));
+                    $totalCount++;
+                    @unlink($file);
```

Comment:
@austinderrick Could you check if the file is writable and can be deleted before saying it is purged and trying to unlink it, and indicate to the user if it could not be deleted? It is currently saying it has purged files that may not have actually been removed.

Reply:
Also, if you don't mind, could you also make the same change to the `utilPurgeThumbs()` method? :)